ui.py

```
@@ -0,0 +1,58 @@
+import tkinter as tk
+import re
+from game import *
```

Ruff (0.11.9)

`from game import *` used; unable to detect undefined names

(F403)